dataland-frontend/src/components/general/RegisterButton.vue

```diff
@@ -23,7 +23,8 @@ export default {
       this.getKeycloakPromise()
         .then((keycloak) => {
           if (!keycloak.authenticated) {
-            return keycloak.register();
+            const url = keycloak.createRegisterUrl({ redirectUri: "/searchtaxonomy" });
```

Comment:
Kann man den Redirect nicht auch in Keycloak spezifizieren?

Reply:
In keycloak spezifiziert man erlaubte weiterleituns-URLS. Die tatsächliche weiterleitungsurl spezifiziert man aber in der Request nach Keycloak (meines verständnisses nach)

Reply:
Ok. Hätte gedacht, dass keycloak das von Haus aus kann, habe aber auch nichts dazu gefunden.